CollAction/Frontend/app/global/MobileNavDraw.tsx

```diff
@@ -12,23 +15,23 @@ interface IMobileNavDrawState {
 function AccountManageNav(props) {
   if (props.isLoggedIn) {
     return <li><a href="/manage">Account</a></li>;
-  }else {
+  } else {
     return null;
   }
  }
 
   function LoginStateNav(props) {
   if (props.isLoggedIn) {
     return <li><button type="button" onClick={() => document.getElementById("logOutBtn").click()}>Logout</button></li>;
-  }else {
+  } else {
     return <li><a href="/account/login">Login</a></li>;
   }
  }
 
 export default class MobileNavDraw extends React.Component<IMobileNavDrawProps, IMobileNavDrawState> {
 
-  constructor () {
-    super();
+  constructor (props?: IMobileNavDrawProps, context?: any) {
```

Comment:
I assume you had to do this because of the React upgrade. I did the same (but without the context parameter) when upgrading for the start project form. You might get merge conflicts when merging.

I think it's a good idea to merge master into this branch first to prevent the branches from falling apart any further :-)